pkg/minikube/constants/constants.go

```diff
@@ -38,6 +38,9 @@ const MiniShiftEnvPrefix = "MINISHIFT"
 // MiniShiftHomeEnv is the environment variable used to change the Minishift home directory
 const MiniShiftHomeEnv = "MINISHIFT_HOME"
 
+const MinSupportedOpenshiftTag = ">1.3.1"
```

Comment:
I think 'MinSupportedOpenshiftVersion' is better, tag might be confused with a git tag.